user-registration-app/src/app/views/live-workshop/live-workshop.component.css

```diff
@@ -0,0 +1,97 @@
+.card {
+  border-radius: 10px;
+  padding:20px;
+  transition: box-shadow 0.3s cubic-bezier(.25,.8,.25,1);
+  background-color: #0093E9;
+  background-image: linear-gradient(160deg, #0093E9 0%, #80D0C7 100%);
+}
+
+.card:hover {
+  box-shadow: 0 14px 28px rgba(0,0,0,0.25), 0 10px 10px rgba(0,0,0,0.22);
+}
+
+.card-image {
+  max-height: 100% !important;
+  overflow: visible;
+  align: center;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
+
+.card-image img {
```

Comment:
The images mostly appear to either be squished or stretched (they don't maintain their aspect ratio), I would look into this article for a fix: https://medium.com/@biancapower/how-to-make-a-rectangle-image-a-circle-in-css-2f392bc9abd3

Reply:
oh yea, I was gonna fix that up over the summer and then totally forgot - thanks for the heads up!!